contrib/go/src/python/pants/contrib/go/tasks/go_fmt_task_base.py

```diff
@@ -35,7 +35,7 @@ def go_fmt_invalid_targets(self, flags):
       if sources:
         args = [os.path.join(self.go_dist.goroot, 'bin', 'gofmt')] + flags + list(sources)
         try:
-          output = subprocess.check_output(args)
+          output = subprocess.check_output(args).decode('utf-8')
```

Comment:
Can simplify this to just `decode()` now that we use Py3, which defaults to `'utf-8'`.
```suggestion
          output = subprocess.check_output(args).decode()
```